src/Octoshift/GithubApi.cs

```diff
@@ -304,5 +305,34 @@ public virtual async Task<bool> RevokeMigratorRole(string org, string actor, str
                 return false;
             }
         }
+
+        public virtual async Task<int> StartArchiveGeneration(string apiUrl, string org, object options)
```

Comment:
I would split this into 2 separate functions so we can abstract away the api payload syntax from our command classes. E.g.

```
public virtual async Task<int> StartMetadataArchiveGeneration(string apiUrl, string org, string repo)
{
    var url = $"{apiUrl}/orgs/{org}/migrations";
    var payload = new {
        repositories = new [] { repo },
        exclude_git_data = true,
        exclude_releases = true,
        exclude_owner_projects = true
    };

    var response = await _client.PostAsync(url,  payload);
    ...
}

Reply:
I really like the suggestion 👍